doc/specs/OOBE-Dialog.md

```diff
@@ -0,0 +1,169 @@
+# PowerToys OOBE Dialog
+
+<img align="right" src="./images/Logo.png" />
+
+- **What is it:** Post-install prompted dialog that walks new users through PowerToys basic functionalities and customizations
+- **Author:** Deondre Davis
+- **Spec Status:** Draft
+
+## 1. Overview
+
+### 1.1. Executive Summary
+
+PowerToys suffers from the lack of a guided post-install experience. Without this, it is virtually impossible for new users to quickly get started with PowerToys due to the need to dig through documentation to understand how to use it. The proceeding document proposes the creation of a post-install prompt that introduces users to the various PowerToys and offers helpful insights and engaging customization options to users who normally would not look through the documentation.
+
+### 1.2. Key Definitions/Concepts
+
+Here are definitions of words and acronyms found throughout this document to ensure clarity:
+
+- **OOBE:** Out of box experience – The users’ initial interactions with the product immediately after installing the product and/or launching the product for the first time.
+
+### 1.3. Goals and Non-Goals
+
+Goals:
+
+- Create a guided prompt that exposes the user to a brief overview of each PowerToy, highlighting the PowerToys’ basic usage.
+- Present a short-list of useful customization options that the user can select to define their experience.
+
+Non-goals:
+
+- Present an exhaustive list of features and options that are available in PowerToys. This OOBE prompt is only meant to provide initial exposure for which interested users should continue investigating via provided links to documentation and settings.
+
+## 2. Definition of Success
+
+### 2.1. Customers
+
+The PowerToys OOBE is for new power users and developers who are looking to tune and streamline their Windows experience for greater productivity and enhanced user experience. As the PowerToys customer base tends to be particularly biased against OOBE prompts in general, we need to present the PowerToys OOBE dialog in such a way that it provides immediate value to end-users to improve the likelihood of users discovering all the PowerToys by completing the prompts.
+
+### 2.2. Expected Impact: Customer, and Technology Outcomes
+
+- **High Reliability:** Less than 0.1% crash rate.
+- **Increased Activation:** 30% or more increase in usage in all PowerToys features.
+- **High User Retention:** 25% or more active PowerToys users after 28 days.
+- **High OOBE Completion Rate:** 50%  or more of New PowerToys users reach the last page of the OOBE prompt.
```

Comment:
If we have the left side navigation pane this might be difficult to achieve.. Maybe something like "User clicks on 3+ items to understand what they are" ?)

Reply:
Hm, that's a good point. This metric is meant to support causation arguments for the Increased Activation metric above, so maybe there's a better measure we can use to track it. I like your suggestion, so we can go with that for now.